sonic-net/sonic-buildimage
# add 200G values to pg profile lookup

This change adds 200G settings for Tomahawk 5 to the pg_profile_lookup.

Fixes: https://github.com/aristanetworks/sonic-qual.msft/issues/159

#### Why I did it

This change is needed in order for some qos tests to run against TH5.

#### How I did it

Added the values then manually ran qos tests identified in the issue above to verify fix.

#### How to verify it

Run qos/test_qos_sai.py::TestQosSai::testParameter and confirm that the IndexError in the issue above does not occur.

#### Which release branch to backport (provide reason below if selected)
